back/courses/models.py

```diff
@@ -1 +1,57 @@
-# Create your models here.
+from courses.models import Course
+from django.db import models
+from news.models import Post
+from social.models import Student
+
+
+class Update(models.Model):
+    date = models.DateTimeField()
+    old_course = models.ForeignKey(Course, on_delete=models.CASCADE)
+    new_course = models.ForeignKey(Course, on_delete=models.CASCADE)
+
+    def __str__(self):
+        return self.old_course.name + " : " + self.new_course.name
+
+
+class Course(models.Model):
+    name = models.CharField(max_length=100, default="Cours inconnu")
```

Comment:
I think it would be better to but "Cours sans nom" or something similar as a default value